source/simulator/parameters.cc

```diff
@@ -126,6 +126,14 @@ namespace aspect
                        "here, one can choose the time step as large as one wants (in particular, "
                        "one can choose $c>1$) though a CFL number significantly larger than "
                        "one will yield rather diffusive solutions. Units: None.");
+
+    prm.declare_entry ("Use smoothing", "false",
```

Comment:
Can you either call this `Use artificial viscosity smoothing`, or maybe better move the corresponding parameter into the section that defines the artificial viscosity parameters?